.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before opening an issue**
+Questions should go to the GEE mailing list (https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/gee-oss)  or Slack Open GEE channel (http://slack.opengee.org/)
+GitHub issues are for bug reports and suggestions for new features.
+
```

Comment:
It would be good to add a break between the each section to maintain clarity between sections

Reply:
Fixed